modules/s3db/deploy.py

```diff
@@ -1193,9 +1212,26 @@ def deploy_rheader(r, tabs=[], profile=False):
                                TH("%s: " % table[f].label, **attr)
                 value = lambda f, table=table, record=record, **attr: \
                                TD(table[f].represent(record[f]), **attr)
+
+                def event_type_link_label():
```

Comment:
Writing functions like this inside makes it hard to read(poor readability).
You could just save it in a variable